packages/core/src/help/help.contents.ts

```diff
@@ -174,10 +174,22 @@ const helpContents: { [key: string]: string } = {
   'pipeline.config.trigger.webhook.source': `
       <p>Determines the target URL required to trigger this pipeline, as well as how the payload can be transformed into artifacts.</p>
   `,
+  'pipeline.config.trigger.cdevents.source': `
+        <p>Determines the target URL required to trigger this pipeline, as well as how the payload can be transformed into artifacts.</p>
+    `,
```

Comment:
Since this is identical to the webhook one, just reuse it. A CDEvent trigger is technically a webhook anyway.

Reply:
yes that's correct, I can reuse the Webhook one.